packages/x-data-grid/package.json

```diff
@@ -35,11 +35,8 @@
   ],
   "scripts": {
     "typescript": "tsc -p tsconfig.json",
-    "build": "pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
-    "build:node": "node ../../scripts/build.mjs node --babel-flag=\"--copy-files\" --babel-flag=\"--no-copy-ignored\"",
-    "build:stable": "node ../../scripts/build.mjs stable --babel-flag=\"--copy-files\" --babel-flag=\"--no-copy-ignored\"",
-    "build:copy-files": "node ../../scripts/copyFiles.mjs",
-    "build:types": "tsx ../../scripts/buildTypes.mts --removeCss",
+    "build": "code-infra build && pnpm build:copy-files",
+    "build:copy-files": "code-infra copy-files --glob \"src/**/*.css\" --glob \"src/**/*.css:esm\"",
```

Comment:
The readme file is not copied to the package anymore:

<img width="874" height="595" src="https://github.com/user-attachments/assets/4dcf1c30-fbab-40e7-b4db-bfe986c6d06f" />

Reply:
Fixed.